streamer/src/nonblocking/quic.rs

```diff
@@ -387,6 +435,7 @@ fn handle_chunk(
 #[derive(Debug)]
 struct ConnectionEntry {
     exit: Arc<AtomicBool>,
+    stake: u64,
```

Comment:
this is kind of meta-issue. but i think we should take epoch boundaries into consideration.

Otherwise, people can temporally stake more and retain the quic connection indefinitely? also consider slashing.

Reply:
Created #26357 to keep track of this.